Makefile

```diff
@@ -1,107 +1,121 @@
-NAME=Libertinus
-VERSION=6.11
-
-DIST=$(NAME)-$(VERSION)
-
-SOURCEDIR=sources
-BUILDDIR=build
-GSUB=$(SOURCEDIR)/features/gsub.fea
-DOC=documentation
-TOOLS=tools
-
-PY?=python3
-BUILD=$(TOOLS)/build.py
-NORMALIZE=$(TOOLS)/sfdnormalize.py
-
-NULL=
-
-FONTS=Sans-Regular \
-       Sans-Bold \
-       Sans-Italic \
-       Serif-Regular \
-       Serif-Semibold \
-       Serif-Bold \
-       Serif-Italic \
-       Serif-SemiboldItalic \
-       Serif-BoldItalic \
-       SerifDisplay-Regular \
-       Math-Regular \
-       SerifInitials-Regular \
-       Mono-Regular \
-       Keyboard-Regular \
-       $(NULL)
-
-SFD=$(FONTS:%=$(SOURCEDIR)/$(NAME)%.sfd)
-NRM=$(FONTS:%=$(BUILDDIR)/$(NAME)%.nrm)
-CHK=$(FONTS:%=$(BUILDDIR)/$(NAME)%.chk)
-OTF=$(FONTS:%=$(NAME)%.otf)
-SVG=preview.svg
-PDF=$(DOC)/Opentype-Features.pdf $(DOC)/Sample.pdf $(DOC)/Math-Sample.pdf
+NAME = Libertinus
+VERSION = 6.11
+
+MAKEFLAGS := -s -j$(shell nproc) -Otarget
```

Comment:
I'd suggest adding a comment to this line explaining why these flags are added.

Reply:
See [comment](https://github.com/alif-type/libertinus/pull/322#discussion_r407460766).